Model/Source/RetourInTheBox.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Magento\Model\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+class RetourInTheBox implements OptionSourceInterface
+{
+    const NOT_ACTIVE        = 'notActive';
+    const NO_OPTIONS        = 'noOptions';
+    const EQUAL_TO_SHIPMENT = 'equalToShipment';
+
+    /**
+     * Options getter
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        return [
+            ['value' => self::NOT_ACTIVE, 'label' => __('No')],
+            ['value' => self::NO_OPTIONS, 'label' => __('No options')],
+            ['value' => self::EQUAL_TO_SHIPMENT, 'label' => __('Equal to shipment')]
+        ];
+    }
+
+    /**
+     * Get options in "key-value" format
+     *
+     * @return array
+     */
+    public function toArray()
+    {
+        return [
+            self::NOT_ACTIVE        => __('No'),
```

Comment:
Kun je een oplossing bedenken waarbij dit niet dubbel in de codes staat?

Reply:
van Richard hoef ik dat niet aan te passen. 😃

Reply:
Maar van mij wel! 💢  

We kunnen het ook samen doen hoor als je geen makkelijke oplossing ziet :P

Reply:
Heb nu iets geprobeerd, ben benieuwd wat je ervan vind